js/msp.js

```diff
@@ -965,8 +963,28 @@ var MSP = {
                 break;
 
             case MSP_codes.MSP_PID_ADVANCED:
-                if (semver.gte(CONFIG.apiVersion, "1.16.0")) {
-                    var offset = 0;
+                var offset = 0;
+                if (semver.gte(CONFIG.flightControllerVersion, "3.0.0")) {
```

Comment:
Everywhere else in this file, `CONFIG.apiVersion` is used for feature gating. Switching to using `CONFIG.flightControllerVersion` will just make it more messy. I can see reasons for removing apiVersions, but if it is done it should be done in a consistent manner, not just here and there.